src/track/cueinfo.h

```diff
@@ -73,4 +73,50 @@ inline bool operator!=(
     return !(lhs == rhs);
 }
 
+inline QDebug operator<<(QDebug debug, const CueType& cueType) {
```

Comment:
I suggest moving these implementations into the .cpp file to keep the header file lean and clean. Performance doesn't matter here.